test/integration/http_tls_enabled_validate_test.lua

```diff
@@ -0,0 +1,116 @@
+local t = require('luatest')
+local fio = require("fio")
+
+local http_server = require('http.server')
+local helpers = require('test.helpers')
+
+local g = t.group()
+
+local ssl_data_dir = fio.pathjoin(helpers.get_testdir_path(), "ssl_data")
+
+local test_cases = {
+    ssl_cert_file_missing = {
+        opts = {
+            ssl_key_file = fio.pathjoin(ssl_data_dir, 'server.key'),
+        },
+        expected_err_msg = "ssl_key_file and ssl_cert_file must be set to enable TLS",
+    },
+    ssl_cert_file_incorrect_type = {
+        opts = {
+            ssl_cert_file = 1,
+        },
+        expected_err_msg = "ssl_cert_file option must be a string",
+    },
+    cert_file_not_exists = {
+        opts = {
+            ssl_key_file = fio.pathjoin(ssl_data_dir, 'server.key'),
+            ssl_cert_file = "some/path",
+        },
+        expected_err_msg = "file \"some/path\" not exists",
```

Comment:
Or alternatively you can use
```lua
expected_err_msg = 'file "some/path" not exists',
```
Up to you.

Reply:
Edited, thank you.